alinka/widget/containers/main_body/content/browser/__init__.py

```diff
@@ -0,0 +1,101 @@
+from PySide6.QtCore import QAbstractTableModel, QModelIndex, Qt
+from PySide6.QtWidgets import (
+    QAbstractItemView,
+    QTableView,
+    QTabWidget,
+    QVBoxLayout,
+    QWidget,
+)
+
+from alinka.db.queries import filter_decisions_by_pesel_child_name
+from alinka.widget.components import LabeledInputComponent
+
+
+class DecisionsTableModel(QAbstractTableModel):
+    filter_by: str | None
+    columns_mapping = {"id": "id", "child_full_name": "Imię i nazwisko dziecka", "child_pesel": "PESEL dziecka"}
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.filter_by = None
+
+    @property
+    def _data(self):
+        decisions = filter_decisions_by_pesel_child_name(self.filter_by)
+        columns_to_include = self.columns_mapping.keys()
+        return [list(decision.model_dump(include=columns_to_include).values()) for decision in decisions]
+
+    def rowCount(self, parent: QModelIndex = QModelIndex()) -> int:
+        return len(self._data)
+
+    def columnCount(self, parent: QModelIndex = QModelIndex()) -> int:
+        return len(self._data[0]) if self._data else 0
```

Comment:
It's not going to break right now, so it's approvable, but instead of `else 0` we could just 
```suggestion
        return len(self.columns_mapping)
```
I've already did something similar in https://github.com/magul/alinka-pyside/blob/develop/alinka/widget/containers/main_body/content/settings/settings_tabs/team_member_tab/team_member_table_group.py#L24-L25 (but there I've just relied on number of columns in SQLAlchemy table)